components/automate-chef-io/content/docs/backup.md

```diff
@@ -183,10 +187,10 @@ If backups are stored in S3:
 chef-automate backup restore s3://bucket_name/path/to/backups/BACKUP_ID
 ```
 
-To restore a backup of an [airgapped installation]({{< relref "airgapped-installation.md" >}}), you must specify the [Airgap Installation Bundle]({{< relref "airgapped-installation.md#create-an-airgap-installation-bundle" >}}) the installation is using:
+To restore a backup of an [airgapped installation]({{< relref "airgapped-installation.md" >}}), you must extract your backup archive into the default backup directory (`/var/opt/chef-automate/backups`) and specify the [Airgap Installation Bundle]({{< relref "airgapped-installation.md#create-an-airgap-installation-bundle" >}}) the installation is using:
```

Comment:
It'd be good to be explicit about distinguishing the single-file archive case from the default.

Reply:
Updated to match your feedback.